src/sql/src/plan/statement/ddl.rs

```diff
@@ -881,7 +893,7 @@ pub fn plan_create_source(
                     .map(|(idx, _type)| idx)
                     .ok_or_else(|| anyhow!("No such column in source key constraint: {}", col))?;
                 if desc.get_unambiguous_name(name_idx).is_none() {
-                    bail!("Ambiguous column in source key constraint: {}", col);
+                    anyhow::bail!("Ambiguous column in source key constraint: {}", col);
```

Comment:
That `anyhow!` a few lines up constrained the error type to `anyhow::Error` rather than `PlanError`. I've adjusted all such examples (there were tons throughout the crate, it turns out) to use a new `sql_err!` macro instead of `anyhow`.